tests/test_mommy.py

```diff
@@ -236,6 +236,20 @@ def test_access_related_name_of_m2m(self):
         except TypeError:
             self.fail('type error raised')
 
+    def test_save_object_instances_when_handling_one_to_many_relations(self):
+        owner = mommy.make(models.Person)
+        dogs_set = mommy.prepare(
+            models.Dog,
+            owner=owner,
+            _quantity=2,
+        )
+        home = mommy.make(
+            models.Home,
+            owner=owner,
+            dogs=dogs_set,
+        )
+        self.assertEqual(home.dogs.count(), 2)
```

Comment:
I did try to run this test without added in `model_mommy/mommy.py` change, it is still green.
Could you, please, try it as well @anapaulagomes ? I might miss something.

I also think, we might need some output from @timthelion to know a bit more about their models structure to be sure we did reproduce the same case here.

Reply:
Thanks @amureki. I took a look again and I noticed a few things. Actually, `dogs_set` shouldn't have an owner. Now the test is fixed. Which leads to the real problem:

> E   ValueError: save() prohibited to prevent data loss due to unsaved related object 'owner'.

I think this is happening because `dogs_set` is using _prepare_ instead of _make_, mixing persisted and not persisted objects. So, the error message seems right (despite pointing to the wrong object). Since I _jumped onto the passing tram_, as we say in Brazilian Portuguese, I thought that this solution was ok but it seems that mommy is working fine. @berinhard we need your input here. Can you help us? :)

Reply:
@berinhard can you say if this test makes sense?